README.md

```diff
@@ -210,6 +227,13 @@ class NewProvider(override val hooks: List<Hook<*>>, override val metadata: Meta
         // add necessary changes on context change
     }
 
+    override fun observe(): Flow<OpenFeatureEvents> {
+        // return the observation flow
+    }
+
+    override fun getProviderStatus(): OpenFeatureEvents {
+        // return the event representative of the current Provider Status
```

Comment:
We should introduce actual [Status types](https://openfeature.dev/specification/types/#provider-status), rather than re-using Events.

Reply:
For sure! Creating an issue that references this.